src/test/resources/data/recipe_book_extension_test/recipes/recipe_book_extension_recipe_5.json

```diff
@@ -0,0 +1,22 @@
+{
+  "type": "recipe_book_extension_test:test_recipe",
+  "conditions" : [
+    {
+      "type" : "forge:false"
```

Comment:
Should be `ModLoadedCondition` instead as otherwise the recipes are not testable.